junebug/__init__.py

```diff
@@ -2,6 +2,28 @@
 Junebug is a system for managing text messaging transports via a RESTful HTTP
 interface.
 '''
+import os
+
+r = os.environ.get('REACTOR')
```

Comment:
I would prefix this with something. Maybe `TWISTED_REACTOR`? Or `JUNEBUG_REACTOR`?